cpu/cc26x0/periph/rfc.c

```diff
@@ -72,13 +58,54 @@ void rfc_irq_disable(void)
     NVIC_EnableIRQ(RF_HW_IRQN);
 }
 
+uint32_t rfc_send_cmd(uint32_t ropAddr)
+{
+    RFC_DBELL->CMDR = ropAddr;
+  /* wait for cmd ack (rop cmd was submitted successfully) */
+  while(RFC_DBELL->RFACKIFG<<31);
+  while(!RFC_DBELL->CMDSTA);
+  if (RFC_DBELL->CMDSTA == 1)
+  {
+    printf("%s: radio operation command acknowledged. \n",__FUNCTION__);
+  }
+  else
+  {
+    printf("%s: acknowledgement of radio operation command failed (0x%" PRIu32 ") \n",__FUNCTION__,RFC_DBELL->CMDSTA);
+  }
+
+  return RFC_DBELL->CMDSTA;
+}
+
+uint16_t rfc_wait_cmd_done(void* ropCmd)
```

Comment:
this function is waiting for asynchronous radio operation commands and prints basic info about `cmd->status` before returning it.

in my opinion it's worth splitting the waiting out: `rfc_wait_for_async_cmd(radio_op_command_t *ropCmd, uint32_t timeout_loop_cnt)`.

What remains is really a `rfc_cmd_get_status` that also prints info to STDIO (which should really be controlled by the application).